### PR TITLE
Bug 1147107 - Update mozilla-download to fetch builds from taskcluster i...

### PR DESCRIPTION
**Pull request by gaye**
...ndex instead of ftp r=lightsofapollo

### CODE REVIEW

File: src/shell.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { exec } from 'child_process';
+
+/**
+ * @fileoverview Thin promise wrapper around ChildProcess#exec.
+ */
+export default function shell(command, options={}) {
```
Comment (lightsofapollo-taskcluster) [inline src/shell.js]: also see the `mz` package which provides promise friendly versions of exec/etc...

File: src/extract.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  })
+  .then(() => {
+    // Copy to destination.
+    return cpr(options.dest, dest);
+  });
+}
+
+function extractDmg(options) {
+  // TODO(gareth)
+  return Promise.reject(new Error('How to dmg?'));
```
Comment (lightsofapollo-taskcluster) [inline src/extract.js]: I wrote another package for this way back that we use in older code IIRC.

File: src/shell.js (hunk 1 above)
Comment (gaye) [inline src/shell.js]: ooo mz

**Comment by lightsofapollo** [comment]
looks good start... let's land this and iterate.

**Pull request merged**